tests/testthat/test-retriever.R

```diff
@@ -1,7 +1,107 @@
-library(testthat)
-library(rdataretriever)
+context("regresion tests")
+library(DBI)
+library('RPostgreSQL')
+
 
 test_that("datasets returns some known values", {
   skip_on_cran()
   expect_identical("car-eval" %in% rdataretriever::datasets(), TRUE)
 })
+
+
+test_that("Download the raw portal dataset into './data/'", {
+  portal <- list("3299474", "3299483", "5603981")
+  rdataretriever::download('portal', './data/')
+  for(file in portal)
+  {
+    file_path <- normalizePath(file.path(getwd(), "tests/testthat/data/", file), winslash = "//", mustWork = FALSE)
+    expect_identical((file.info(file_paths)$size == 0)==FALSE, TRUE)
+  }
+})
+
+
+test_that("Install the portal into csv", {
+  # Install portal into csv files in your working directory
+  portal <- list("portal_main", "portal_plots", "portal_species")
+  rdataretriever::install('portal', 'csv')
+  for(file in portal)
+  {
+    file_path <- normalizePath(file.path(getwd(), "tests/testthat/", paste(file, ".csv")), winslash = "//", mustWork = FALSE)
```

Comment:
Add newlines before `winslash` and `mustWork` to keep line length reasonable. (same below and probably above for consistency)